src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+                BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
```

Comment:
<img width="1080" alt="image" src="https://github.com/mate-academy/jv-work-with-file/assets/81781479/92574928-ee59-486c-8db1-e1b9d1d0c72b">


```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(data, toFileName);
```